work.css

```diff
@@ -7,7 +7,7 @@
   margin: 0;
   padding: 0;
   box-sizing: border-box;
-  font-family: 'Poppins' sans-serif;
+  font-family: 'Poppins', sans-serif;
 }
 
 img {
```

Comment:
Also, please as mentioned in the project's requirement stick to the Figma as much as possible. Let's add a background image from the .css file for the contact form section (not `img` tag from .html file). Also, add the curve at the left top corner for the contact section. And change the name of the button `Submit` to `Get In Touch`.

| Your Design | Figma Design |
|-------------|---------------|
| <img width="400" alt="image" src="https://user-images.githubusercontent.com/34578631/165386016-7cac0132-8627-4f77-8065-e9cf7d3660d5.png"> | <img width="500" alt="image" src="https://user-images.githubusercontent.com/34578631/165385802-977ad3ac-13e7-4c9a-b960-ab5b8af26f33.png"> |